### ISSUE TITLE
Motion Discriminator classifier 

### ISSUE THREAD

**Issue by ikvision** [mkocabas/VIBE#45]
**Is your feature request related to a problem? Please describe.**
I am not clear how  the motion discriminator classify fake/true sequence

**Describe the solution you'd like**
The output seems to be an unnormalized vector of size 2 for each sequence. I was expecting a softmax layer to use it for a classification decision
https://github.com/mkocabas/VIBE/blob/731c27382978e242fc1492c4bf53ddd72fd345de/lib/models/motion_discriminator.py#L78

**Additional context**
Maybe Its usage in the training loss

**Comment by mkocabas** [comment]
Hi @ikvision,

We follow LSGAN and they do not perform normalization. 

Check this reference [implementation](https://github.com/eriklindernoren/PyTorch-GAN/blob/master/implementations/lsgan/lsgan.py#L96) and the Figure 4 in the paper: https://arxiv.org/abs/1611.04076
